lib/services/saml.go

```diff
@@ -49,18 +49,18 @@ func ValidateSAMLConnector(sc types.SAMLConnector, rg RoleGetter) error {
 	if sc.GetEntityDescriptorURL() != "" {
 		resp, err := http.Get(sc.GetEntityDescriptorURL())
 		if err != nil {
-			return trace.Wrap(err)
+			return trace.WrapWithMessage(err, "unable to fetch entity descriptor from %q for SAML connector:%v", sc.GetEntityDescriptorURL(), sc.GetName())
```

Comment:
```suggestion
			return trace.WrapWithMessage(err, "unable to fetch entity descriptor from %v for SAML connector %v", sc.GetEntityDescriptorURL(), sc.GetName())
```

I don't think we need to quote the URL, and let's add a space after the word connector.

(Same for quoting the URL below too)

Reply:
ok updated

Reply:
Thanks, all url output updated to `%v`